tests/testsuite/features.rs

```diff
@@ -109,8 +109,17 @@ fn invalid4() {
 
     assert_that(p.cargo("build"),
                 execs().with_status(101).with_stderr("\
-[ERROR] Package `bar v0.0.1 ([..])` does not have these features: `bar`
-"));
+error: failed to select a version for `bar`.
+    ... required by package `foo v0.0.1 ([..])`
+versions that meet the requirements `*` are: 0.0.1
+
+the package `bar` depends on `bar`, with features: `bar` but it does not have these features.
```

Comment:
Hm shouldn't this read "the package foo" instead of "the package bar"?

Reply:
Yes, and it is fixed. Thanks. (Sorry about all the force pushing.)